proxy/logging/unit-tests/benchmark_LogObject.cc

```diff
@@ -0,0 +1,169 @@
+/** @file
```

Comment:
How does this get built into an executable?  I can't see it in any makefile changes.

Reply:
I took it out of the build since I couldn't figure out how to make it not break the ci build.

Reply:
I concur, there's not much value in having benchmarks in the CI runs.  Can you add a short comment that explains how to build it as a stand-alone executable?